packages/composables/src/_test/useSetup.ts

```diff
@@ -8,6 +8,7 @@ type Injections = {
   apiClient: {
     invoke: ReturnType<typeof vi.fn>;
   };
+  [key: string]: unknown;
```

Comment:
why do we need this generic injections? Would be better to know exactly what we're dealing with